README.md

```diff
@@ -57,11 +57,11 @@ size = moab.size
 #### Validate if Moab Object is Well-Formed
 
 ```ruby
-moab = Moab::StorageObject.new(druid, object_dir)
+moab = Moab::StorageObject.new(object, object_dir)
```

Comment:
maybe `object_id` instead of `object`?  this makes me think it'll be a more complex instantiated object, but it's a string id, right?

good catch on not making it `druid` in the non-stanford case, though... maybe we should have another PR that makes that change across this whole README?

Reply:
Agreed that `object_id` is clearer. Made that change, and ticketed https://github.com/sul-dlss/moab-versioning/issues/87